src/Microsoft.ML.Core/Data/SynchronizedCursorBase.cs

```diff
@@ -9,8 +9,7 @@ namespace Microsoft.ML.Data
     /// <summary>
     /// Base class for creating a cursor on top of another cursor that does not add or remove rows.
     /// It forces one-to-one correspondence between items in the input cursor and this cursor.
-    /// It delegates all ICursor functionality except Dispose() to the root cursor.
-    /// Dispose is virtual with the default implementation delegating to the input cursor.
```

Comment:
The Dispose() method of SynchronizedCursorBase used to be virtual, but now it isn't, so guess it's safe to remove this last line.

Reply:
In the base class `DataViewRow` it is virtual, and classes deriving from `SynchronizedCursorBase` can still override it (and some of them do).

---
In reply to: [368201483](https://github.com/dotnet/machinelearning/pull/4673#discussion_r368201483) [](ancestors = 368201483)